src/app/components/hero-action/hero-action.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-hero-action',
+  templateUrl: './hero-action.component.html',
+  styleUrls: ['./hero-action.component.scss']
+})
+export class HeroActionComponent implements OnInit {
+  private _events: string[] = ['Love and die', 'Fuck and drink', 'Live and kill while sacrificing everything'];
```

Comment:
keep this variable empty. if there is no battle event, this should be shown, so keep this array empty. :)

Reply:
solved